modules/cnf-intel-n3000-optimize-data-plane.adoc

```diff
@@ -0,0 +1,45 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-understanding-intel-hardware-accelertor-card-for-openshift-container-platform_{context}"]
+= Understanding Intel hardware accelerator cards for OpenShift Container Platform
+
+Hardware accelerator cards from Intel® accelerates 4G/LTE and 5G Virtualized Radio Access Networks (vRAN) workloads, increasing in turn the overall compute capacity of a commercial, off-the-shelf platform.
```

Comment:
s/accelerates/accelerate/, right?

I also think you have two sentences.  I'm leery of suggesting an alternative because I am unsure how the hardware processes packets to increase the compute capacity.